premake5.lua

```diff
@@ -312,8 +312,8 @@
 			links       { "ole32", "ws2_32", "advapi32", "version" }
 			files { "src/**.rc" }
 
-		filter "toolset:mingw"
-			links		{ "crypt32", "bcrypt" }
+		filter { "system:windows", "toolset:not msc" }
```

Comment:
What about LLVM with Visual Studio? How should this be handled?

Reply:
So big build matrix!

I honestly don't know
- if clang under visual should have the link
- if clang under mingw should have the link.
It seems to be `usage` from curl.

I can use `filter { "system:windows", "toolset:gcc" }` to not change those configs

Reply:
This feels like a bigger change like we discussed, does this solve an immediate problem?

Reply:
Discussion was adding extra API which would be a big change IMO.
Here I (propose to) remove the strange toolset mingw (which is no even web documented)